packages/base-map/i18n/en-US.yml

```diff
@@ -0,0 +1,30 @@
+otpUi:
+  mapControls:
+    AttributionControl:
```

Comment:
Do we use all of these settings, and therefore do we need to support i18n for all of these?